spec/lib/bamboozled/tabular_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../../spec_helper'
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.